shared/uavobjectdefinition/multirotorairframesettings.xml

```diff
@@ -0,0 +1,44 @@
+<xml>
+	<object name="MultirotorAirframeSettings" singleinstance="true" settings="true" category="Platform">
+		<description>Select airframe type.  Currently used by @ref ActuatorModule to choose mixing from @ref ActuatorDesired to @ref ActuatorCommand</description>
+		
+		<field name="MultirotorType" units="" type="enum" elements="1" options="Tri,QuadP,QuadX,HexaP,HexaX,HexaCoax,Octo,OctoV,OctoCoaxP,OctoCoaxX" defaultvalue="QuadX"/>
+		
+		<field name="MotorChannel" units="" type="enum" defaultvalue="None">
```

Comment:
So just starting my comments reading here, apologies if this is out of context. Similarly to the current approach of storing the UI information, this mapping is just to cue in the system. There is nothing in code that forces this and the mixer to be consistent, right?